projects/netgrif-components/src/lib/navigation/group-navigation-component-resolver/default-components/default-tabbed-case-view/default-tabbed-case-view.component.ts

```diff
@@ -79,4 +123,75 @@ export class DefaultTabbedCaseViewComponent extends AbstractTabbedCaseViewCompon
         }, this._autoswitchToTaskTab, this._openExistingTab);
     }
 
+    protected openTab(openCase: Case) {
+        this._injectedTabData.tabViewRef.openTab({
+            label: {
+                text: openCase.title,
+                icon: openCase.icon ? openCase.icon : 'check_box'
+            },
+            canBeClosed: true,
+            tabContentComponent: this._injectedTabData.tabViewComponent,
+            injectedObject: {
+                baseFilter: this.resolveFilter(openCase),
+                allowedNets: this.resolveAllowedNets(openCase),
+                navigationItemTaskData: this._injectedTabData.navigationItemTaskData,
+                searchTypeConfiguration: this._injectedTabData.taskViewSearchTypeConfiguration,
+                showMoreMenu: this._injectedTabData.taskViewShowMoreMenu,
+                headersChangeable: this._injectedTabData.taskViewHeadersChangeable,
+                headersMode: this._injectedTabData.taskViewHeadersMode,
+                allowTableMode: this._injectedTabData.taskViewAllowTableMode,
+                defaultHeadersMode: this._injectedTabData.taskViewDefaultHeadersMode
+            },
+            order: this._injectedTabData.tabViewOrder,
+            parentUniqueId: this._injectedTabData.tabUniqueId
+        }, this._autoswitchToTaskTab, this._openExistingTab);
+    }
+
+    private resolveFilter(openCase: Case): Filter {
+        const additionalFilter = this._injectedTabData.taskViewAdditionalFilter;
+        const mergeFilters = this._injectedTabData.taskViewMergeWithBaseFilter;
+        const baseFilter = new SimpleFilter('', FilterType.TASK, {case: {id: `${openCase.stringId}`}});
+
+        let filter;
+        if (additionalFilter === undefined) {
+            filter = baseFilter;
```

Comment:
If ``additionalFilter`` is ``undefined``, you can directly return ``baseFilter``, so your ``if``statement can be simplified as follows.
```
if (additionalFilter === undefined) {
     return baseFilter;
}
if (mergeFilters) {
     return additionalFilter.merge(baseFilter, MergeOperator.AND);
}
return additionalFilter;
```